### PR TITLE
FPCommand form: Only display enabled verification/event commands

### PR DESCRIPTION
**Pull request by mistydemeo**
Previously the filtering would display _all_ of these commands, which means every older revision would be in the dropdown.

Fixes #6512.

### CODE REVIEW

File: fpr/forms.py [hunk 1]
```diff
@@ -137,8 +137,8 @@ class FPCommandForm(forms.ModelForm):
     def __init__(self, *args, **kwargs):
         super(FPCommandForm, self).__init__(*args, **kwargs)
 
-        verification_commands = fprmodels.FPCommand.objects.filter(command_usage='verification')
-        event_detail_commands = fprmodels.FPCommand.objects.filter(command_usage='event_detail')
+        verification_commands = fprmodels.FPCommand.objects.filter(command_usage='verification', enabled=1)
```
Comment (Hwesta) [inline fpr/forms.py]: `enabled=True` is preferable, since enabled is a BooleanField

**Comment by mistydemeo** [comment]
Updated to use `enabled=True`.

**Comment by Hwesta** [comment]
Great, :shipit:

**Comment by mistydemeo** [comment]
Realized `FPCommand.active.filter` covers this perfectly well, so updated.

**Pull request closed without merging**